pkg/api/addons.go

```diff
@@ -926,6 +926,18 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 		}
 	}
 
+	// Ensure cloud-node-manager is enabled on appropriate upgrades for Azure Stack cloud
+	if isUpgrade &&
+		cs.Properties.IsAzureStackCloud() &&
+		to.Bool(cs.Properties.OrchestratorProfile.KubernetesConfig.UseCloudControllerManager) &&
+		common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.21.0") {
+		// Force enabling cloud-node-manager addon
+		log.Infoln("Updating cloud-node-manager addon to 'true' on Azure Stack cloud...")
```

Comment:
I am not sure this message is adding relevant information given that `upgrade.go` is already logging something similar.

If you see extra value, then I would use the `debug` level instead of `info`

Reply:
unnecessary log removed